packages/react-i18n/src/i18n.ts

```diff
@@ -333,7 +333,6 @@ export class I18n {
     return this.getCurrencySymbolLocalized(this.locale, currency);
   };
 
-  @memoize((currency: string, locale: string) => `${locale}${currency}`)
```

Comment:
this is no longer required as our `getCurrencySymbol()` below is performing the memoization